.github/workflows/osx.yml

```diff
@@ -0,0 +1,32 @@
+name: OSX-build
+
+on: [push]
+
+jobs:
+  osx-clang-omp:
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+        - {shared: "ON", build_type: "Debug", name: "omp/debug/shared"}
+        - {shared: "OFF", build_type: "Release", name: "omp/release/static"}
+    name: ${{ matrix.config.name }}
+    runs-on: [macos-latest]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup
+      run: brew install libomp
```

Comment:
Does AppleClang really ship without OpenMP?

Reply:
Yes, without this the server does not find OpenMP.